experiments/causal_pipeline_resample/bash_scripts/run_multiple_experiments.sh

```
@@ -0,0 +1,543 @@
+            echo "DEBUG: Adding timeout-factor to args: $TIMEOUT_FACTOR"
+        fi
+
+        # Add data path arguments
+        EXPERIMENT_ARGS="$EXPERIMENT_ARGS --meds \"$MEDS_DATA\""
+        EXPERIMENT_ARGS="$EXPERIMENT_ARGS --features \"$FEATURES_DATA\""
+        EXPERIMENT_ARGS="$EXPERIMENT_ARGS --tokenized \"$TOKENIZED_DATA\""
+        EXPERIMENT_ARGS="$EXPERIMENT_ARGS --pretrain-model \"$PRETRAIN_MODEL\""
+
+        # Start heartbeat process to keep terminal active during experiment
+        (
+            local elapsed=0
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)